Migrations/MyPlacesDbContext.cs

```diff
@@ -49,7 +49,21 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         entity.Property(d => d.DonutShopName).IsRequired();
         entity.Property(d => d.DonutShopAddress);
         entity.Property(d => d.DonutShopWebsite);
-        entity.Property(d => d.DonutShopImage);
+        entity.Property(d => d.DonutShopCityState);
+        entity.HasData(
+            new { DonutShopId = 1, DonutShopName = "Five Daughters Bakery", DonutShopAddress = "1900 Eastland Ave Suite 101", DonutShopWebsite = "https://fivedaughtersbakery.com/", DonutShopCityState = "Nashville TN"},
+            new { DonutShopId = 2, DonutShopName = "Heavenly Donuts", DonutShopAddress = "1313 Shasta St", DonutShopWebsite = "https://www.heavenlydonut.com/", DonutShopCityState = "Redding CA"},
```

Comment:
I know it's silly, but can we add a comma in the city/state for all of these? 
"Nashville, TN"

Reply:
I guess?  Not sure why it matters?

Reply:
I know it's a tiny detail, but that's proper grammar. :)